TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

```diff
@@ -34,13 +33,13 @@
 import com.qualcomm.hardware.rev.RevHubOrientationOnRobot;
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.DcMotorEx;
-import com.qualcomm.robotcore.hardware.DcMotorSimple;
```

Comment:
This is needed for reversing encoders and motors

Reply:
Yes, I keep this unused import around to make things a little easier for teams.

Reply:
I put it back :)

Reply:
I still see it removed -- is Android Studio optimizing imports on the fly?

Reply:
I think so, considering I explicitly typed the import out